tests/futures/test_futures_base_api.py

```diff
@@ -6,8 +6,10 @@
 """Module that checks the general Futures Base API class."""
 
 from asyncio import run
+from unittest import IsolatedAsyncioTestCase
 
 import pytest
+from proxy import TestCase
```

Comment:
Is that the usage of the new dependency "proxy.py"?

Reply:
Correct. This is a class that extends `unittest.TestCase` that automatically sets up and tears down the proxy.